assets/prototype/application/services/config/useConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { ConfigContext } from '../context/ConfigProvider';
+
+type returnType = {
+	dateFormat: string;
+	timeFormat: string;
+	timezone: string;
+};
```

Comment:
This is a bit of safety for destructuring.

Reply:
![image](https://user-images.githubusercontent.com/1477453/70901063-b8c28280-2002-11ea-9e3f-a8c75cd9a794.png)